Source/Core/DiscIO/Volume.h

```diff
@@ -11,9 +11,8 @@
 #include <string>
 #include <vector>
 
-#include <mbedtls/sha1.h>
-
 #include "Common/CommonTypes.h"
+#include "Common/Crypto/SHA1.h"
```

Comment:
Could `Context` be forward declared?

Reply:
Is there a real advantage? The header is purposefully small and doesn’t expose internals